### ISSUE TITLE
Quote Block: Keyboard focus behavior differs from theme to theme

### ISSUE THREAD

**Issue by t-hamano** [WordPress/gutenberg#46546]
### Description

When certain keystrokes are performed in a quote block, the behavior differs depending on the theme.

The quate block displays the block inserter button because the content consists of inner blocks. When the arrow-up key is pressed after the cite element has been given focus, its behavior varies depending on the theme.

Essentially, the first keystroke focuses on the block inserter and the second keystroke focuses on the innner block.

<details>
<summary>Screencast on Twenty Twenty Two</summary>

https://user-images.githubusercontent.com/54422211/207631520-4eab3cea-17c7-44d0-ba8b-8ff95720988d.mp4

</details>

Curiously, however, in Twenty Twenty Three, the focus is skipped on the block inserter and given to the inner block.

<details>
<summary>Screencast on Twenty Twenty Three</summary>

https://user-images.githubusercontent.com/54422211/207632055-16bf6816-e403-4acb-9439-9d3af7423264.mp4

</details>

I further investigated and found that this problem occurs when **the cite element doesn't overlap the inserter**.

<details>
<summary>Screencast on Twenty Twenty Three</summary>

https://user-images.githubusercontent.com/54422211/207632055-16bf6816-e403-4acb-9439-9d3af7423264.mp4

</details>

I don't know if this is a problem with the block editor or the writing flow, or just expected behavior of the browser, but I am reporting it as an issue.

### Step-by-step reproduction instructions

- Activate Twenty Twenty Three theme.
- Insert a quote block.
- Focus the cite element.
- Press the arrow-up key.
- Confirm that the block inserter disappears and the inner block comes into focus.

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes

**Comment by ellatrix** [comment]
This is probably fixed after #60307?

**Comment by t-hamano** [comment]
#60307 seems to have removed the appender entirely, but when I select a block inside the Quote block, the quick inserter still appears in the top right. Is this intended?

Either way, I think this issue can be closed.

![quote-block-inserter](https://github.com/user-attachments/assets/3d205078-a26b-4ffb-ac7a-fdfcebaae88b)

**Comment by ellatrix** [comment]
It's not just when selecting in list view, happens too without it. This is the empty default block inserter, it seems to be mispositioned. Let's rename this issue and fix it.

**Comment by ellatrix** [comment]
Or let's open a new issue

**Comment by ellatrix** [comment]
Done in #63517
